themes/scss/map/dashboard-info.scss

```diff
@@ -93,9 +91,9 @@
 }
 .CDB-Dashboard-infoDescription {
   color: #FFF;
-  font-size: $sFontSize-normal;
+  font-size: $sFontSize-medium;
```

Comment:
Name of variable `sFontSize-medium` should be written in all lowercase letters with hyphens instead of underscores